### PR TITLE
Fix/codecov report

### PR DESCRIPTION
**Pull request by Maasdamind**
# How was this tested?

* Step one etc.

# Type of change

Please check all boxes that apply to this change below.

| Type                     | Applies ✅|
| ------------------------ | :-------: |
| Backend                  |           |
| Frontend                 |           |
| Github/Ci/Admin          |     ✅     |
| Documentation required   |           |

# Checklist

Please follow the checklist below and check all completed items as you work on the PR.

I have..
- [ ] performed a self-review of my own code
- [ ] commented my code, particularly in hard-to-understand areas
- [ ] written documentation
- [ ] added automated tests
- [ ] merged dependencies to `develop

### CODE REVIEW

File: .github/workflows/code-coverage.yml [hunk 1]
```diff
@@ -15,9 +14,11 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
       - name: Test code
         run: dotnet test /p:CollectCoverage=true /p:CoverletOutputFormat=opencover
       - name: Upload code
         run: |
           ls
```
Comment (pizzaluc) [inline .github/workflows/code-coverage.yml:23]: `ls` needs to be deleted, was needed for debugging but does nothing

**Comment by Maasdamind** [comment]
@pizzaluc thanks for the heads up. I've also added a scheduled daily run on develop

**Comment by Maasdamind** [comment]
> # [Codecov](https://codecov.io/gh/SimCorp/nats-topology-visualiser/pull/46?src=pr&el=h1) Report
> > ❗ No coverage uploaded for pull request base (`develop@82a9930`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> > The diff coverage is `n/a`.
> 
> > ❗ Current head [4eb0f73](https://github.com/SimCorp/nats-topology-visualiser/commit/4eb0f7331b051cddaff59a89e94aa1233213d63b) differs from pull request most recent head [e3b5143](https://github.com/SimCorp/nats-topology-visualiser/commit/e3b51438504654743f71852b8fe9c0c981a76ef6). Consider uploading reports for the commit [e3b5143](https://github.com/SimCorp/nats-topology-visualiser/commit/e3b51438504654743f71852b8fe9c0c981a76ef6) to get more accurate results
> > [Impacted file tree graph](https://codecov.io/gh/SimCorp/nats-topology-visualiser/pull/46?src=pr&el=tree)
> 
> ```diff
> @@            Coverage Diff            @@
> ##             develop     #46   +/-   ##
> =========================================
>   Coverage           ?   0.00%           
> =========================================
>   Files              ?      15           
>   Lines              ?     531           
>   Branches           ?      26           
> =========================================
>   Hits               ?       0           
>   Misses             ?     531           
>   Partials           ?       0           
> ```
> 
> [Continue to review full report at Codecov](https://codecov.io/gh/SimCorp/nats-topology-visualiser/pull/46?src=pr&el=continue).
> 
> > **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> > `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> > Powered by [Codecov](https://codecov.io/gh/SimCorp/nats-topology-visualiser/pull/46?src=pr&el=footer). Last update [82a9930...e3b5143](https://codecov.io/gh/SimCorp/nats-topology-visualiser/pull/46?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

This still appears since develops commit hash isn't uploaded yet

**Review by pizzaluc — APPROVED** [review]
👍

**Pull request merged**